logging_handler_test.go

```diff
@@ -101,9 +102,13 @@ func TestLoggingHandler_PingUserAgent(t *testing.T) {
 			opts.RawRedirectURL = "localhost"
 			validation.Validate(opts)
```

Comment:
I've started altering these to:

```
err := validation.Validate(opts)
assert.NoError(t, err)
```

Wherever I saw them -- otherwise nil point deference panics were creeping in that were much harder to triage the issue quickly.

And my IDE yells at me whenever I try to commit with an unhandled error 😄

Reply:
Will replace some of them as I go through, don't want to expand the scope of the PR too much though so will only do it near where I've already changed stuff, can make this iterative